### DISCUSSION TITLE
Snapshot rotation

### DISCUSSION THREAD

**Discussion by mach7** [mrlt8/docker-wyze-bridge#726] [Q&A]
Is it possible to rotate a snapshot image?

something like http://localhost:5000/img/camera-nickname.jpg?rotate-right

Or maybe a variable in the Home Assistant Addon config? I'd like to rotate my doorbell snapshot

**Comment by mrlt8** [comment]
Not right now as it will just capture a frame from the video feed as is. However, you should be able to rotate the video using `ROTATE_DOOR=True` which is also available in HA.
